extensions/wikidata/src/org/openrefine/wikidata/qa/EditInspector.java

```diff
@@ -124,7 +123,6 @@ public void inspect(List<ItemUpdate> editBatch, WikibaseSchema schema) throws Ex
         // so that all newly created entities appear in the batch
         SchemaPropertyExtractor fetcher = new SchemaPropertyExtractor();
         Set<PropertyIdValue> properties = fetcher.getAllProperties(schema);
-        List<EntityDocument> propertyDocuments = entityCache.getMultipleDocuments(properties.stream().collect(Collectors.toList()));
```

Comment:
This was added by @darecoder to prefetch property documents in one API call rather than requesting them one by one. I assume your IDE detected that the `propertyDocuments` variable was not used later on. I think we should just call `getMultipleDocuments` without storing the return result in a local variable. It would be good to have a comment here explaining why we are doing that.

Reply:
Got it!